### ISSUE TITLE
[Bug] sponsorblock doesn't appear in the player

### ISSUE THREAD

**Issue by gototheskinny** [qnblackcat/uYouPlus#771]
### Device info
- iOS/iPadOS version: ios 15.0
- Device model:iPhone 12 64
- Sideload tool: TrollStore
- The specific version of uYouPlus: v17.48.2

### Describe the bug
Sponsorblock is not showing up in the player but works.

### How to reproduce the bug
![image](https://user-images.githubusercontent.com/51786709/207997033-61e4bbb8-bb1e-47d3-aadd-a94e17250b15.png)
### Are you using the newest version of uYouPlus? If not, why?
Yes I am using the latest version 

### Does the issue happen with the official YouTube from AppStore?
Official one doesn't have sponsorblock...

### Additional context: 
Earlier versions was not perfect too but this version for me at least is the worst for sponsorblock.

**Comment by yodaluca23** [comment]
Did you try turning on the iSponcerBlock buttons on the video page setting in the iSponsorBlock settings?

**Comment by gototheskinny** [comment]
> Did you try turning on the iSponcerBlock buttons on the video page setting in the iSponsorBlock settings?

Yeah obviously I did
![image](https://user-images.githubusercontent.com/51786709/207998456-0241dc10-c622-468f-973f-69db7ba07fd6.jpeg)

**Comment by dayanch96** [comment]
it could be PoomSmarts last isponsorblock update, i left same message in his fork but no response. Try to downgrade your iSponsorblock to fix it

**Comment by qnblackcat** [comment]
@PoomSmart any idea?

**Comment by PoomSmart** [comment]
Now that I think about it. Has this ever worked on sideloaded?

**Comment by dayanch96** [comment]
> Now that I think about it. Has this ever worked on sideloaded?

sure thing. always

**Comment by PoomSmart** [comment]
Fixed in my fork of iSponsorBlock.

**Comment by arichorn** [comment]
Thanks PoomSmart!

**Comment by gototheskinny** [comment]
> Fixed in my fork of iSponsorBlock.

Thank you!

**Comment by dayanch96** [comment]
still have weird glitch, is it only for me? buttons appears only after some manipulations @qnblackcat 


https://user-images.githubusercontent.com/38832025/208588436-c6a0e26d-8633-4dbb-ba45-52008018eacf.MOV

**Comment by gototheskinny** [comment]
> still have weird glitch, is it only for me? buttons appears only after some manipulations @qnblackcat 
> 
> 
> 
> 
> 
> https://user-images.githubusercontent.com/38832025/208588436-c6a0e26d-8633-4dbb-ba45-52008018eacf.MOV
> 
> 
> 
> 

I hope they solve this issue too.

**Comment by PoomSmart** [comment]
Fork updated.

**Comment by dayanch96** [comment]
> Fork updated.

nothing changed. let me describe this glitch:

when tweak icons for overlay enabled, it doesnt appears properly by the opening, it requires to switch app to the background and open it again so icons can be loaded;
when tweak icons disabled in preferences, it works as should - they are disabled.

**Comment by PoomSmart** [comment]
Weird enough I tested my fork of iSB on my jailbroken iPad and couldn't reproduce the issue.

**Comment by dayanch96** [comment]
@PoomSmart I use it on jailed iPhone 11. Seems like it depends on the screen resolution which app trying to detect.

Now I just turned on ‘iPad style on iPhone’ tweak and it these buttons appears correctly but only in landscape mode

**Comment by qnblackcat** [comment]
Please try [v17.49.6-2.1](https://github.com/qnblackcat/uYouPlus/releases/tag/v17.49.6-2.1)

**Comment by dayanch96** [comment]
> Please try [v17.49.6-2.1](https://github.com/qnblackcat/uYouPlus/releases/tag/v17.49.6-2.1)

same problem, same workarounds

https://user-images.githubusercontent.com/38832025/209471642-5fd5f8e7-4d8f-457a-b9c9-70175e79886a.MOV

**Comment by gototheskinny** [comment]
> Please try [v17.49.6-2.1](https://github.com/qnblackcat/uYouPlus/releases/tag/v17.49.6-2.1)

I've tried and it works well for me. Installed without deleting the previous version BTW.

**Comment by gototheskinny** [comment]
I don't wanna be annoying so I I am submitting this issue in here.

There is a new issue in the latest version. When u try to submit a segment sponsorblock ui doesn't work like in this vid:
[github doesn't let me upload this file... so here it is](https://www.veed.io/view/83f302a4-5711-44bc-918a-616250f52a5b?sharingWidget=true&panel=share)

**Comment by dayanch96** [comment]
@qnblackcat @gototheskinny i was doing multilingual version months ago and just updated to actual version just to fix yt crashes and seems like it works like a charm


https://user-images.githubusercontent.com/38832025/210134508-da256348-ee32-4890-be86-97e9d3563bc6.MOV

here is fork: https://github.com/dayanch96/iSponsorBlock

**Comment by qnblackcat** [comment]
Please update to [v17.49.6-2.3~1](https://github.com/qnblackcat/uYouPlus/releases/tag/v17.49.6-2.3(1))

**Comment by gototheskinny** [comment]
> Please update to [v17.49.6-2.3~1](https://github.com/qnblackcat/uYouPlus/releases/tag/v17.49.6-2.3(1))

Appairing issue is fixed now but [this](https://github.com/qnblackcat/uYouPlus/issues/771#issuecomment-1365480917) issue is still there. The overlay ui of sponsorblock isn't working.

**Comment by qnblackcat** [comment]
> > Please update to [v17.49.6-2.3~1](https://github.com/qnblackcat/uYouPlus/releases/tag/v17.49.6-2.3(1))
> 
> Appairing issue is fixed now but [this](https://github.com/qnblackcat/uYouPlus/issues/771#issuecomment-1365480917) issue is still there. The overlay ui of sponsorblock isn't working.

see #854

**Comment by gototheskinny** [comment]
> > > Please update to [v17.49.6-2.3~1](https://github.com/qnblackcat/uYouPlus/releases/tag/v17.49.6-2.3(1))
> 
> > 
> 
> > Appairing issue is fixed now but [this](https://github.com/qnblackcat/uYouPlus/issues/771#issuecomment-1365480917) issue is still there. The overlay ui of sponsorblock isn't working.
> 
> 
> 
> see #854

OK. Thank you for fixing this issue!

**Comment by qnblackcat** [comment]
> OK. Thank you for fixing this issue!

Nope. PoomSmart is the real hero here

**Comment by dayanch96** [comment]
btw, is that thing is chip bar?

![8B0D0B6A-4A11-4DF1-AA46-0FE4F387C93A](https://user-images.githubusercontent.com/38832025/211139655-bd9fa2f4-d9f2-409e-8495-816e11174ccd.jpeg)

it never appears in the latest update. bool disabled
